pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -285,16 +341,39 @@ func (c *clustersReconciler) updateStatus(ctx context.Context, cluster *kueuealp
 	return c.localClient.Status().Update(ctx, cluster)
 }
 
+func (c *clustersReconciler) runGC(ctx context.Context) {
+	log := ctrl.LoggerFrom(ctx)
+	if c.gcInterval == 0 {
+		log.V(2).Info("Garbage Collection is disabled")
+		return
+	}
+	log.V(2).Info("Starting Garbage Collector")
+	for {
+		select {
+		case <-ctx.Done():
+			log.V(2).Info("Garbage Collector Stopped")
+			return
+		case <-time.After(c.gcInterval):
+			log.V(5).Info("Run Garbage Collection")
```

Comment:
```suggestion
			log.V(4).Info("Run Garbage Collection for Lost Remote Workloads")
```
Nit, following the log level from core k8s: https://github.com/kubernetes/kubernetes/blob/23728378380ec853676c4d5bf7b39ae8dbc7a74f/pkg/controller/podgc/gc_controller.go#L147

Reply:
done